src/util/sh.ts

```diff
@@ -0,0 +1,20 @@
+import { exec } from "child_process";
+
+/**
+ * Executes a shell script and returns the output
+ *
+ * Sample usage: sh("npx npm run build")
+ */
+export function sh(command: string): Promise<string> {
```

Comment:
We already have a util for this: https://github.com/vivid-planet/comet-upgrade/blob/main/src/util/execute-command.util.ts